eth/chains/base.py

```diff
@@ -667,11 +667,72 @@ def import_block(self,
         self.header = self.ensure_header()
         return result
 
+    def mine_all(
+            self,
+            transactions: Sequence[SignedTransactionAPI],
+            *args: Any,
+            parent_header: BlockHeaderAPI = None,
+            **kwargs: Any,
+    ) -> Tuple[BlockImportResult, Tuple[ReceiptAPI, ...], Tuple[ComputationAPI, ...]]:
+
+        if parent_header is None:
+            base_header = self.header
+        else:
+            base_header = self.create_header_from_parent(parent_header)
+
+        vm = self.get_vm(base_header)
+
+        new_header, receipts, computations = vm.apply_all_transactions(transactions, base_header)
+        filled_block = vm.set_block_transactions(vm.get_block(), new_header, transactions, receipts)
+
+        block_result = vm.mine_block(filled_block, *args, **kwargs)
+        imported_block = block_result.block
+
+        self.validate_block(imported_block)
+
+        # TODO: extract to _import_validated_block (and push to parent class)
```

Comment:
Just double checking if the comment was deliberately left here or if you wanted to address this / file an issue.

Reply:
Cool, yeah, resolved.